src/Utilities.js

```diff
@@ -0,0 +1,22 @@
+
+// Implementation of https://developer.mozilla.org/en-US/docs/Web/API/Element/closest
+const findClosest = (el, selector) => {
+  while ((el = el.parentElement) && !el.matches(selector)) {
+    // Intentionally empty.
+  }
+  return el
+}
+
+const isNestedInTable = (el) => {
+  while ((el = el.parentElement)) {
```

Comment:
As mentioned in the comment above iirc these parenthesis actually are required to evaluate the assignment. Will double-check and add tests for this method too.